components/sections/ChatMessagingArea.tsx

```diff
@@ -113,13 +141,24 @@ const Header = (props: {
           <ModalBody>
             <Stack>
               {/* TODO: handle error situation of no listing or ID (shouldn't happen?) */}
-              <NextLink href={listingRouteBuilder(listing?.id)} passHref>
-                <Link>
-                  <ButtonSecondary isFullWidth>View listing</ButtonSecondary>
-                </Link>
-              </NextLink>
-              {/* TODO: implement or remove */}
-              <ButtonSecondary isFullWidth>Accept tenant</ButtonSecondary>
+              <Tooltip label="test">
+                <NextLink href={listingRouteBuilder(listing?.id)} passHref>
+                  <Link>
+                    <ButtonSecondary isFullWidth>View listing</ButtonSecondary>
+                  </Link>
+                </NextLink>
+              </Tooltip>
+              <Tooltip
+                isDisabled={!requestDisabled}
+                hasArrow
+                label={disabledRequestErrorMsg}
+              >
+                <Box>
+                  <ButtonSecondary isFullWidth isDisabled={requestDisabled}>
+                    Send Tenant Request
+                  </ButtonSecondary>
```

Comment:
Not sure if you intended to do this with this PR, but did you want to let tenants use this button to send tenant requests?

If so, it looks like you need to use the `handleTenantRequest` you made. If not, it's also ok to leave it here or do something else since you already covered it in the listing contact card part (and that idk if the extra work is worth it :sweat_smile: ).

Reply:
I'll work on implementing it! Oops forgot to do that

Reply:
Added caf8716